index.html

```diff
@@ -0,0 +1,681 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://g1.nyt.com/fonts/css/web-fonts.5810def60210a2fa7d0848f37e3fa048bb6147b1.css" rel="stylesheet"
+        type="text/css">
+    <script src="https://kit.fontawesome.com/d4de0f4540.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun - The New York Times</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div id="nav_bar">
+                <h2 id="ny_logo"><a
+                        href="https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0"
+                        target="_blank">The New York Times</a></h2>
+                <ul class="left_nav">
+                    <li><a href="#" class="nav_icon fas fa-bars"></a></li>
```

Comment:
Kindly use ```<button>``` instead of ```<a href="#" class="nav_icon fas fa-bars">```
![hamb](https://user-images.githubusercontent.com/46388113/74051154-5e74eb00-49cf-11ea-989c-d282529f4c03.PNG)